Source/Scene/ClassificationModel.js

```diff
@@ -0,0 +1,1869 @@
+     *
+     * @demo {@link http://cesiumjs.org/Cesium/Apps/Sandcastle/index.html?src=3D%20Models.html|Cesium Sandcastle Models Demo}
+     */
+    function ClassificationModel(options) {
+        options = defaultValue(options, defaultValue.EMPTY_OBJECT);
+
+        var gltf = options.gltf;
+        if (gltf instanceof ArrayBuffer) {
+            gltf = new Uint8Array(gltf);
+        }
+
+        if (gltf instanceof Uint8Array) {
+            // Binary glTF
+             gltf = parseBinaryGltf(gltf);
+        } // else Normal glTF (JSON)
+
+        this._gltf = gltf;
+
+        this._basePath = defaultValue(options.basePath, '');
+        var baseUri = getBaseUri(document.location.href);
+        this._baseUri = joinUrls(baseUri, this._basePath);
+
+        /**
+         * Determines if the model primitive will be shown.
+         *
+         * @type {Boolean}
+         *
+         * @default true
+         */
+        this.show = defaultValue(options.show, true);
+
+        /**
+         * The 4x4 transformation matrix that transforms the model from model to world coordinates.
+         * When this is the identity matrix, the model is drawn in world coordinates, i.e., Earth's WGS84 coordinates.
+         * Local reference frames can be used by providing a different transformation matrix, like that returned
+         * by {@link Transforms.eastNorthUpToFixedFrame}.
+         *
+         * @type {Matrix4}
+         *
+         * @default {@link Matrix4.IDENTITY}
+         *
+         * @example
+         * var origin = Cesium.Cartesian3.fromDegrees(-95.0, 40.0, 200000.0);
+         * m.modelMatrix = Cesium.Transforms.eastNorthUpToFixedFrame(origin);
+         */
+        this.modelMatrix = Matrix4.clone(defaultValue(options.modelMatrix, Matrix4.IDENTITY));
+        this._modelMatrix = Matrix4.clone(this.modelMatrix);
+
+        this._ready = false;
+        this._readyPromise = when.defer();
+
+        /**
+         * This property is for debugging only; it is not for production use nor is it optimized.
+         * <p>
+         * Draws the bounding sphere for each draw command in the model.  A glTF primitive corresponds
+         * to one draw command.  A glTF mesh has an array of primitives, often of length one.
+         * </p>
+         *
+         * @type {Boolean}
+         *
+         * @default false
+         */
+        this.debugShowBoundingVolume = defaultValue(options.debugShowBoundingVolume, false);
+        this._debugShowBoundingVolume = false;
+
+        /**
+         * This property is for debugging only; it is not for production use nor is it optimized.
+         * <p>
+         * Draws the model in wireframe.
+         * </p>
+         *
+         * @type {Boolean}
+         *
+         * @default false
+         */
+        this.debugWireframe = defaultValue(options.debugWireframe, false);
+        this._debugWireframe = false;
+
+        this._distanceDisplayCondition = options.distanceDisplayCondition;
```

Comment:
Remind me where this is used via 3D Tiles?

Reply:
I thought it was used, but was not. Removed.